bpf-metrics-exporter/README.md

```diff
@@ -14,24 +14,32 @@ to correlate process IDs -> containers -> k8s pods.
 
 The following metrics are exported:
 
-- `bpf_program_size_jitted_bytes`: The size of the BPF program in bytes.
-- `bpf_program_size_translated_bytes`: The size of the BPF program in bytes.
-- `bpf_program_mem_bytes`: The amount of memory used by the BPF program in bytes.
-- `bpf_program_verified_instructions_total`: The number of instructions in the BPF program.
-
-
-The following will be added pending: https://github.com/open-telemetry/opentelemetry-rust/issues/1242
-
-- `bpf_programs_total`: The number of BPF programs loaded into the kernel.
-
-Labels:
-
-- `id`: The ID of the BPF program
-- `name`: The name of the BPF program
-- `type`: The type of the BPF program
-- `tag`: The tag of the BPF program
-- `gpl_compatible`: Whether the BPF program is GPL compatible
-- `load_time`: The time the BPF program was loaded
+- `bpf_programs`: Information on each loaded BPF Program
+    - Labels:
+        - `id`: The ID of the BPF program
+        - `name`: The name of the BPF program
+        - `type`: The type of the BPF program as a readable string
+        - `tag`: The tag of the BPF program
+        - `gpl_compatible`: Whether the BPF program is GPL compatible
+        - `load_time`: The time the BPF program was loaded
+- `bpf_maps`: Information of each loaded BPF Map
+    - Labels:
+        - `id`: The ID of the BPF map
+        - `name`: The name of the BPF map
+        - `type`: The type of the BPF map as an `u32` which corresponds to the following [kernel enumeration](https://elixir.bootlin.com/linux/v6.6.3/source/include/uapi/linux/bpf.h#L906)
+        - `key_size`: The key size in bytes for the BPF map
+        - `value_size`: The value size for the BPF map
+        - `max_entries`: The maximum number of entries for the BPF map.
+        - `flags`: Loadtime specific flags for the BPF map
+- `bpf_links`: Information on each of the loaded BPF Link
+    - Labels:
+        - `id`: The ID of the bpf Link
+        - `prog_id`: The Program ID of the BPF program which is using the Link.
+        - `type`: The BPF Link type as a `u32` which corresponds to the following [kernel enumeration](https://elixir.bootlin.com/linux/v6.6.3/source/include/uapi/linux/bpf.h#L1048)
+- `bpf_program_size_jitted_bytes_total`: The total size of all loaded BPF programs in bytes.
+- `bpf_program_size_translated_bytes_total`: The total size of all loaded BPF programs in bytes.
+- `bpf_program_mem_bytes_total`: The total amount of memory used by all loaded BPF programs in bytes.
+- `bpf_program_verified_instructions_total`: The total number of instructions in all loaded BPF programs.
```

Comment:
Please revert these changes to the metrics that were here.

Reply:
Done